tox.ini

```diff
@@ -142,7 +142,12 @@ commands =
     #bash -c "set -e && cd {toxinidir}/examples/basic_usage && bash usage.sh"
     # Disable slow training at runtime for ase example to speed up CI for now.
     #bash -c "set -e && cd {toxinidir}/examples/ase && bash train_export.sh"
-    sphinx-build {posargs:-E} -W -b html docs/src docs/build/html
+    sphinx-build \
+        {posargs:-E} \
+        --fail-on-warning \
+        --doctree-dir {temp_dir}/.doctrees \
```

Comment:
A tempdir means this will can't be cached and doc build might take longer. What I did on metatensor was to move this to `docs/build/doctrees`, which is then not copied to the website (only `docs/build/html` is)